src/System.Net.WebSockets.Client/tests/project.json

```diff
@@ -22,6 +22,7 @@
     "netstandard1.3": {}
   },
   "supports": {
+    "coreFx.Test.netcore50": {},
```

Comment:
This is what is causing the failure, I repro this locally with diagnostics log:

```
System.Net.Security 4.3.0 provides a compile-time reference assembly for System.Net.Security on UAP,Version=v10.0, but there is no run-time assembly compatible with win10-x86.
    One or more packages are incompatible with UAP,Version=v10.0 (win10-x86).
    System.Net.Security 4.3.0 provides a compile-time reference assembly for System.Net.Security on UAP,Version=v10.0, but there is no run-time assembly compatible with win10-x86-aot.
    One or more packages are incompatible with UAP,Version=v10.0 (win10-x86-aot).
    System.Net.Security 4.3.0 provides a compile-time reference assembly for System.Net.Security on UAP,Version=v10.0, but there is no run-time assembly compatible with win10-x64.
    One or more packages are incompatible with UAP,Version=v10.0 (win10-x64).
    System.Net.Security 4.3.0 provides a compile-time reference assembly for System.Net.Security on UAP,Version=v10.0, but there is no run-time assembly compatible with win10-x64-aot.
    One or more packages are incompatible with UAP,Version=v10.0 (win10-x64-aot).
```

This tests depend on System.Net.Security but apparently that dependency doesn't support netcore50. https://github.com/dotnet/corefx/blob/release/1.1.0/src/System.Net.Security/pkg/System.Net.Security.pkgproj#L18-L20

cc: @weshaggard who has more context in the infrastructure for this older branches.